tmp/lanelet2_extension/lib/query.cpp

```diff
@@ -212,6 +212,35 @@ std::vector<lanelet::NoStoppingAreaConstPtr> query::noStoppingAreas(
   return no_reg_elems;
 }
 
+std::vector<lanelet::NoParkingAreaConstPtr> query::noParkingAreas(
+  const lanelet::ConstLanelets & lanelets)
+{
+  std::vector<lanelet::NoParkingAreaConstPtr> no_pa_reg_elems;
+
+  for (const auto & ll : lanelets) {
+    std::vector<lanelet::NoParkingAreaConstPtr> ll_no_pa_re =
+      ll.regulatoryElementsAs<lanelet::autoware::NoParkingArea>();
+
+    // insert unique tl into array
```

Comment:
I think this is originally copied from L137, but needs to be updated.
```suggestion
    // insert unique NoParkingArea into array
```